README.md

```diff
@@ -40,21 +40,26 @@ $ age-plugin-tpm --generate -o age-identity.txt
 $ age-plugin-tpm -y age-identity.txt > age-recipient.txt
 
 # Encrypt / Decrypt something
-$ echo "Hack The Planet" | age -R ./age-recipient.txt -o test-decrypt.txt
-$ age --decrypt -i ./age-identity.txt -o - test-decrypt.txt
+$ echo 'Hack The Planet!' | age -R age-recipient.txt -o test-decrypt.txt
+$ age --decrypt -i age-identity.txt -o - test-decrypt.txt
 Hack The Planet!
 ```
 
-### With PIN
+You can add `--pin` when calling `--generate` to require a PIN when encrypting or decrypting.
```

Comment:
Alternatively, we could make the default example use `--pin`, which is preferred, and maybe an addendum in place of this text that says, you can opt to not use `--pin`.